test/sql/setops/test_setops.test

```diff
@@ -115,3 +115,13 @@ SELECT 1, 'a' UNION ALL SELECT 1, 'a' UNION SELECT 2, 'b' UNION SELECT 1, 'a' OR
 2	b
 1	a
 
+
+# EXCEPT ALL / INTERSECT ALL
+query II
```

Comment:
Perhaps it makes sense to copy some of [Postgres union test cases](https://github.com/postgres/postgres/blob/7606175991f8ed5ae36eecf6951cb89dcfb2156e/src/test/regress/sql/union.sql)? We've already extracted the tables before in the [lateral join test cases](https://github.com/duckdb/duckdb/blob/1f04c207eb67711acd89265fba210c57716d9f3f/test/sql/subquery/lateral/pg_lateral.test#L4).

Reply:
will add those tests thanks